tools/cli/domainCommands.go

```diff
@@ -353,6 +353,38 @@ func (d *domainCLIImpl) failoverDomains(c *cli.Context) ([]string, []string) {
 	return succeedDomains, failedDomains
 }
 
+// RebalanceDomains is used for managed failover to rebalance with domain data PreferredCluster
+func (d *domainCLIImpl) RebalanceDomains(c *cli.Context) {
+	// ask user for confirmation
+	prompt("You are trying to rebalance all managed domains, continue? Y/N")
```

Comment:
A dry run option may be useful here? so that operator can have an understand of the result of the rebalance without actually doing it.

Reply:
nit: maybe don't print this prompt in dry run mode, so that operation know he's in dry run mode and there'll be no actual impact.